libbeat/publisher/queue/memqueue/broker.go

```diff
@@ -56,6 +56,7 @@ type Settings struct {
 	FlushMinEvents int
 	FlushTimeout   time.Duration
 	WaitOnClose    bool
+	IntQueueSize   int
```

Comment:
My preference would be to use `InternalQueueSize`, since that's what's used in the earlier structure and "Int" as an abbreviation is overloaded with "Integer"